src/style.css

```diff
@@ -1 +1,67 @@
 /* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 12px;
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  width: 100%;
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  list-style: none;
+}
+
+.nav__link {
+  height: 60px;
+  white-space: nowrap;
+  display: flex;
+  align-items: center;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #000;
+  font-weight: 500;
+  cursor: pointer;
+}
+
+.nav__item + .nav__item {
+  margin-left: 20px;
+}
+
+.nav__link:hover,
+.nav__link.is-active {
```

Comment:
Don't bind rules together
Because then you will have several places to check to get all the rules for the same element (as you have it here with the next rule)